src/fides/api/ops/api/v1/endpoints/user_permission_endpoints.py

```diff
@@ -109,6 +123,7 @@ async def get_user_permissions(
                 id=CONFIG.security.oauth_root_client_id,
                 user_id=CONFIG.security.oauth_root_client_id,
                 scopes=SCOPE_REGISTRY,
+                roles=[ADMIN],
```

Comment:
For this root client, I'm adding the admin role in addition to all the scopes - either avenue gives the root user access to all the endpoints.   In case the user workflow is later deprecated to just assigning roles directly, I thought we should start assigning roles now.